src/Models/Scopes/Product/WithProductAttributesScope.php

```diff
@@ -26,7 +26,8 @@ public function apply(Builder $builder, Model $model)
             ->addSelect($builder->getQuery()->from . '.entity_id AS id')
             ->addSelect($builder->getQuery()->from . '.sku')
             ->addSelect($builder->getQuery()->from . '.visibility')
-            ->addSelect($builder->getQuery()->from . '.type_id AS type');
+            ->addSelect($builder->getQuery()->from . '.type_id AS type')
+            ->addSelect($builder->getQuery()->from . '.created_at');
```

Comment:
OR nicer might be
```suggestion
            ->addSelect([
                'id' => $model->qualifyColumn('entity_id')
                'type' => $model->qualifyColumn('type_id')
            ])
            ->addSelect($model->qualifyColumns([
                'sku',
                'visibility'
                $this->getCreatedAtColumn()
            ]));
```

Reply:
I'm not sure on this one, it feels harder to read the code when it's split apart like this. It's less immediately obvious what's being selected in what way.

Reply:
Agree with @Jade-GG